### PR TITLE
Feature/storage

### PR DESCRIPTION
**Pull request by Ayushi-knoldus**

### CODE REVIEW

File: pallets/carpooling/src/lib.rs [hunk 1]
```diff
@@ -34,10 +50,17 @@ pub mod pallet {
...
+	#[pallet::storage]
+    #[pallet::getter(fn get_driver)]
+    pub type Driver<T: Config> = StorageMap<_, Blake2_128Concat, T::AccountId, DriverOf<T>>;
+	#[pallet::storage]
+    #[pallet::getter(fn get_booking)]
+    pub type Booking<T: Config> = StorageMap<_, Blake2_128Concat, CustomerOf<T>, DriverOf<T>>;
 	// Learn more about declaring storage items:
 	// https://substrate.dev/docs/en/knowledgebase/runtime/storage#declaring-storage-items
-	pub type Something<T> = StorageValue<_, u32>;
+	//pub type Something<T> = StorageValue<_, u32>;
```
Comment (aman2909verma) [inline pallets/carpooling/src/lib.rs:63]: remove this comment

File: pallets/carpooling/src/lib.rs [hunk 2]
```diff
@@ -74,15 +97,15 @@ pub mod pallet {
...
+			//<Something<T>>::put(something);
 
 			// Emit an event.
 			Self::deposit_event(Event::SomethingStored(something, who));
 			// Return a successful DispatchResultWithPostInfo
 			Ok(())
 		}
 
-		/// An example dispatchable that may throw a custom error.
+		/*/// An example dispatchable that may throw a custom error.
```
Comment (aman2909verma) [inline pallets/carpooling/src/lib.rs:108]: do not comment out this dispatch call, delete it instead

File: pallets/carpooling/src/lib.rs [hunk 3]
```diff
@@ -34,10 +50,17 @@ pub mod pallet {
...
 	#[pallet::storage]
-	#[pallet::getter(fn something)]
+	#[pallet::getter(fn get_customer)]
+    pub type Customer<T: Config> = StorageMap<_, Blake2_128Concat, T::AccountId, CustomerOf<T>>;
+	#[pallet::storage]
+    #[pallet::getter(fn get_driver)]
+    pub type Driver<T: Config> = StorageMap<_, Blake2_128Concat, T::AccountId, DriverOf<T>>;
+	#[pallet::storage]
+    #[pallet::getter(fn get_booking)]
+    pub type Booking<T: Config> = StorageMap<_, Blake2_128Concat, CustomerOf<T>, DriverOf<T>>;
```
Comment (aman2909verma) [inline pallets/carpooling/src/lib.rs:60]: change this StorageMap to map from T::AccountId to T::AccountId

**Review by aman2909verma — CHANGES_REQUESTED** [review]

**Review by aman2909verma — APPROVED** [review]

File: pallets/carpooling/src/lib.rs [hunk 4]
```diff
@@ -18,8 +18,24 @@ mod benchmarking;
 #[frame_support::pallet]
 pub mod pallet {
 	use frame_support::{dispatch::DispatchResult, pallet_prelude::*};
-	use frame_system::pallet_prelude::*;
-
+	use frame_system::{Account, pallet_prelude::*};
+
+	type CustomerOf<T> = SCustomer<<T as frame_system::Config>::Hash>;
```
Comment (bishtpawan) [inline pallets/carpooling/src/lib.rs:23]: Please add documentation

File: pallets/carpooling/src/lib.rs [hunk 5]
```diff
@@ -74,31 +96,12 @@ pub mod pallet {
 			let who = ensure_signed(origin)?;
 
 			// Update storage.
-			<Something<T>>::put(something);
+			//<Something<T>>::put(something);
```
Comment (bishtpawan) [inline pallets/carpooling/src/lib.rs:99]: please remove commented code

**Comment by bishtpawan** [comment]
Raise PR for dev branch

**Pull request closed without merging**